_aggregations/metric/sum.md

```diff
@@ -9,49 +9,199 @@ redirect_from:
+
+```json
+GET /deliveries/_search
+{
+  "size": 0,
+  "aggs": {
+    "total_weight": {
+      "sum": {
+        "field": "weight_kg"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response contains value of `45.6` corresponding to the sum of `12.5` + `7.8` + `15.0` + `10.3`:
+
+```json
+{
+  ...
+  "hits": {
+    "total": {
+      "value": 4,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "total_weight": {
+      "value": 45.6
+    }
+  }
+}
+```
+
+### Using a script to compute values
+
+Instead of specifying a field directly, you can provide a script to calculate values for the aggregation. This is useful when the value must be derived or adjusted.
+
+In the following example, each weight is converted from kilograms to grams before summing using a script:
 
 ```json
-GET opensearch_dashboards_sample_data_ecommerce/_search
+GET /deliveries/_search
 {
   "size": 0,
   "aggs": {
-    "sum_taxful_total_price": {
+    "total_weight_grams": {
       "sum": {
-        "field": "taxful_total_price"
+        "script": {
+          "source": "doc['weight_kg'].value * 1000"
+        }
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
-#### Example response
+The response includes `total_weight_grams` of `45600`:
 
 ```json
 {
-  "took": 16,
-  "timed_out": false,
-  "_shards": {
-    "total": 1,
-    "successful": 1,
-    "skipped": 0,
-    "failed": 0
+  ...
+  "hits": {
+    "total": {
+      "value": 4,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
   },
+  "aggregations": {
+    "total_weight_grams": {
+      "value": 45600
+    }
+  }
+}
+```
+
+### Combining a field with a value script
+
+You can also specify both a `field` and a `script`, using the special variable `_value` to reference the field’s value. This is useful when applying transformations to existing field values.
+
+The following example increases all weights by 10% before summing:
+
+```json
+GET /deliveries/_search
+{
+  "size": 0,
+  "aggs": {
+    "adjusted_weight": {
+      "sum": {
+        "field": "weight_kg",
+        "script": {
+          "source": "Math.round(_value * 110) / 100.0"
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response reflects a 10% increase applied to the original total weight.
+
+```json
+{
+  ...
   "hits": {
     "total": {
-      "value": 4675,
+      "value": 4,
       "relation": "eq"
     },
     "max_score": null,
     "hits": []
   },
   "aggregations": {
-    "sum_taxful_total_price": {
-      "value": 350884.12890625
+    "adjusted_weight": {
+      "value": 50.16
+    }
+  }
+}
+```
+
+### Handling missing values
+
+Documents missing the target field are ignored by default. To include them using a default value, use the `missing` parameter. This ensures that documents without the field are treated as having zero weight and included in the aggregation.
```

Comment:
It's zero weight in this example, correct? You can set it to any value? I would clarify this.